src/Bicep.Core/Emit/EmitLimitationVisitor.cs

```diff
@@ -42,6 +45,21 @@ protected override void VisitInternal(SyntaxBase node)
             this.stack.Pop();
         }
 
+        private ExpressionKind GetExpressionKind(SyntaxBase node)
+        {
+            if (node is FunctionCallSyntax && 
```

Comment:
The condition here duplicates the logic in `ExpressionEmitter`. However, the copy of the logic here will be removed with #479, so it should be ok.